scripts/testing/update_iso

```diff
@@ -95,21 +95,21 @@ def check_input_iso_available():
     If not, notify the user and exit.
     """
     if os.path.exists(INPUT_ISO):
-        print("** using input boot ISO: %s" % INPUT_ISO)
+        print("** using input ISO: %s" % INPUT_ISO)
     else:
-        print("** error: input boot ISO (%s) not found" % INPUT_ISO)
+        print("** error: input ISO (%s) not found" % INPUT_ISO)
         sys.exit(1)
 
 def get_boot_iso_revision():
-    """Check if we have a Git revision for the input boot.iso.
+    """Check if we have a Git revision for the input ISO.
```

Comment:
Looking about it, lets be clear that this is the Anaconda revision we are talking about:

"check if we have an Anaconda Git revision"

Otherwise it could get easily confused with say ostree revisions or even image checksums.